war/pom.xml

```diff
@@ -632,9 +632,9 @@ THE SOFTWARE.
         </configuration>
       </plugin>
       <plugin>
-        <groupId>org.eclipse.jetty</groupId>
-        <artifactId>jetty-maven-plugin</artifactId>
-        <version>10.0.20</version>
+        <groupId>org.eclipse.jetty.ee8</groupId>
+        <artifactId>jetty-ee8-maven-plugin</artifactId>
+        <version>12.0.8</version>
```

Comment:
`jetty:run` Mojo fails with:

> [ERROR] Failed to execute goal `org.eclipse.jetty.ee8:jetty-ee8-maven-plugin:12.0.8:run` (`default-cli`) on project `jenkins-war`: Unable to parse configuration of mojo `org.eclipse.jetty.ee8:jetty-ee8-maven-plugin:12.0.8:run` for parameter `config`: Cannot create instance of class `org.eclipse.jetty.util.resource.Resource`: InstantiationException -> [Help 1]

Likely needs to adapt to https://github.com/jetty/jetty.project/pull/10077.

Reply:
Fixed in commit d137f956b7